include ChartPath in the parameters for upgrade (#11837)
MicrosoftDocs/azure-devops-docs: docs/pipelines/tasks/deploy/helm-deploy.md

<table><thead><tr><th>Parameters</th><th>Description</th></tr></thead>
<tr><td><code>command</code><br/>(Command)</td><td>(Required) Select a helm command.<br/>Default value: ls</td></tr>
<tr><td><code>chartType</code><br/>(Chart Type)</td><td>(Required) Select how you want to enter chart information. You can provide either the name of the chart or the folder/file path to the chart.<br/>Available options: Name, FilePath. Default value: Name</td></tr>

> <tr><td><code>chartName</code><br/>(Chart Name)</td><td>(Required if <code>chartType</code> is <code>Name</code>) Chart reference to install, this can be a url or a chart name. For example, if chart name is <b>stable/mysql</b>, the task will run <b>helm install stable/mysql</b></td></tr>
> <tr><td><code>chartPath</code><br/>(Chart Path)</td><td>(Required if <code>chartType</code> is <code>FilePath</code>) Path to the chart to install. This can be a path to a packaged chart or a path to an unpacked chart directory. For example, if <b>./redis</b> is specified the task will run <b>helm upgrade ./redis</b>. If you are consuming a chart which is published as an artifact then the path will be <b>$(System.DefaultWorkingDirectory)/ARTIFACT-NAME/Charts/CHART-NAME</b></td></tr>

Reviewer: ```suggestion
<tr><td><code>chartName</code><br/>(Chart Name)</td><td>(Required if <code>chartType</code> is <code>Name</code>) Chart reference to install, this can be a URL or a chart name. For example, if chart name is <b>stable/mysql</b>, the task will run <b>helm install stable/mysql</b></td></tr>
<tr><td><code>chartPath</code><br/>(Chart Path)</td><td>(Required if <code>chartType</code> is <code>FilePath</code>) Path to the chart to install. This can be a path to a packaged chart or a path to an unpacked chart directory. For example, if <b>./redis</b> is specified, the task will run <b>helm upgrade ./redis</b>. If you're consuming a chart that's published as an artifact, then the path will be <b>$(System.DefaultWorkingDirectory)/ARTIFACT-NAME/Charts/CHART-NAME</b></td></tr>
```
